src/components/mixins/InputDebounce.js

```diff
@@ -33,17 +36,20 @@ export default {
       if (useDebounce) {
         this.touched = true;
         this.updateValue(valueToEmit);
-      }
-      else {
+      } else {
         this.touched = false;
-        this.$emit('input', valueToEmit);
-        this.$emit('update:value', valueToEmit);
+        this.$emit("input", valueToEmit);
+        this.$emit("update:value", valueToEmit);
       }
     },
-    updateValue: debounce(function (value) {
-      this.touched = false;
-      this.$emit('input', value);
-      this.$emit('update:value', value);
-    }, 500, {'leading': true, 'trailing': false})
+    updateValue: debounce(
+      function (value) {
+        this.touched = false;
+        this.$emit("input", value);
+        this.$emit("update:value", value);
+      },
+      500,
+      { leading: true, trailing: true }
```

Comment:
Here's the main code change. Set `trailing` to be `true`
Here is the Lodash docs https://lodash.com/docs/4.17.15#debounce

The rest is the Eslint / Prettier linting changes :D